spec.md

```diff
@@ -144,18 +144,51 @@ circuit MyTop...
 
 ## Circuits
 
-All FIRRTL circuits consist of a list of modules, each representing a hardware
-block that can be instantiated. The circuit must specify the name of the
-top-level module.
+A FIRRTL circuit is a named collection of FIRRTL modules.  Each module is a
+hardware "unit" that has ports, registers, wires, and may instantiate other
+modules (see: [@sec:modules]).  (This is the same concept as a Verilog
+`module`{.verilog}.)  A public module is usable outside of the current circuit.
+
+Consider the following circuit named `Foo`.  This contains two modules, `Bar`
+and `Baz`.  Module `Baz` is marked public.
 
 ``` firrtl
-circuit MyTop :
-  module MyTop :
-    ; ...
-  module MyModule :
-    ; ...
+circuit Foo :
+  module Bar :
+    input a: UInt<1>
+    output b: UInt<1>
+
+    connect b, a
+
+  public module Bar :
+    input a: UInt<1>
+    output b: UInt<1>
+
+    inst bar of Bar
+    connect bar.a, a
+    connect b, bar.b
+```
+
+A circuit may contain public modules that instantiate no common instances.
+
+In the circuit `Qux` below, there are two public modules, `Corge` and `Grault`.
+Neither instantiates each other:
```

Comment:
```suggestion
Neither instantiates the other:
```
? I don't know but "neither.. each other" seems strange.   It's likely I'm wrong, whatever you think is best :+1:.

Reply:
Made this example longer and this language is no longer present. Good catch.